pkg/errors/codes/codes_string.go

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
+package codes
+
+import (
+	"fmt"
+	"strconv"
+)
+
+// String returns the string representation of the MocCode
+func (c MocCode) String() string {
```

Comment:
Use this in the old GetErrorCode() function for consistency.

Reply:
Any idea where `GetErrorCode()` used?

Reply:
Closing to help me organize PR revisions, feel free to re-open if I didn't address something.